doc/README

```diff
@@ -191,6 +191,8 @@ Hbase - 0.96.1.1-hadoop2+, 0.98.0-hadoop2+, 0.98.1-hadoop2+,
         1.1.6+, 1.1.7+, 1.1.8+, 1.1.9+, 1.2.0+, 1.2.1+, 1.2.2+,
         1.2.3+, 1.2.4+, 1.2.5+, 1.2.6+, 1.3.0+, 1.3.1+
 
+Hive -  1.2.2, 2.1.1, 2.3.0 [HiveNote]
```

Comment:
for this round 1 patch, perhaps we should just stick to one version.  Then we can add support for other versions as we build up a testsuite for Hive.